loan-payoff/REPWRITERSPECS/BANNO.LOAN.PAYOFF.V1.POW

```diff
@@ -191,17 +201,19 @@ SETUP
 
  VALIDCHRINPUT1="32-126,128-175,181-184,189,190,198,199,208-216,221,222,224-253"
 
- PROGRAMVERSION="1.1.0"
- LASTMODDATE='08/04/22'
- LASTMODTIME="14:00 MT"
+ PROGRAMVERSION="1.1.1"
+ LASTMODDATE='06/18/24'
+ LASTMODTIME="17:00 PT"
  PRINTCONFIGINFO=FALSE
- PRINTCONFIGINFODAYS=60
- PROGRAMUPDATENOTE1="Updated CFG LF code so defaults properly handle blank input settings,"
- PROGRAMUPDATENOTE2="and refined check for same loan types in different categories like home,auto,etc."
+ PRINTCONFIGINFODAYS=90
+ PROGRAMUPDATENOTE1="Updated the loggingErrorMessage to include ineligible loan type"
+ PROGRAMUPDATENOTE2="and warning codes, and added Program Installation Date parameter to CFG."
  FOR TMPLOOP=1 TO INDENTMAX
   DO
    INDENT(TMPLOOP)=REPEATCHR(" ",TMPLOOP*2)
   END
+ IF SYSTEMDATE<LASTMODDATE+PRINTCONFIGINFODAYS THEN
```

Comment:
I'm thinking it might be a good idea to move this logic into DISPLAYCONFIGINFO procedure just so all that logic is contained in one place.  Maybe something like this:
```
 IF SYSTEMDATE<LASTMODDATE+PRINTCONFIGINFODAYS THEN
  PRINTCONFIGINFO=TRUE

 IF PRINTCONFIGINFOPARAM=FALSE THEN
  PRINTCONFIGINFO=FALSE
 ELSE
  DO
   IF PRINTCONFIGINFOPARAM=TRUE OR
     (INSTALLDATE<>DATENULL AND
      INSTALLDATE+PRINTCONFIGINFODAYS>SYSTEMDATE) THEN
    PRINTCONFIGINFO=TRUE
  END

```

Reply:
The change will be in my next commit, today.